lib/sablon/content.rb

```diff
@@ -173,8 +173,9 @@ def append_to(paragraph, display_node, env)
     end
 
     # Handles reading image data and inserting it into the document
-    class Image < Struct.new(:name, :data, :local_rid)
+    class Image < Struct.new(:name, :data, :local_rid, :properties)
```

Comment:
Here is the implementation I was thinking of  `class Image < Struct.new(:name, :data,, :properties)` and then replace `attr_accessor :properties` with `attr_accessor :local_rid`. While it is technically bad form to subclass a Struct like we do here it does provide the benefit of automatic attribute accessors for the symbols defined above.

Reply:
Ah now I get it - done